mlabs/src/Mlabs/NFT/Validation.hs

```diff
@@ -312,19 +312,19 @@ mkTxPolicy !datum' !act !ctx =
               && traceIfFalse "Not all used tokens are returned." checkTokenReturned
               && traceIfFalse "Returned Token UTXOs have mismatching datums." checkMissMatchDatumMint
           BuyAct {..} ->
-            traceIfFalse "Transaction cannot mint." noMint
+            traceIfFalse "Transaction cannot mint." True -- noMint TODO: allow minting Gov
```

Comment:
Can we please open issues for each of the todo's.

Reply:
I've commented out this checks so I can run traces, but I think that #269 will implement all of them